cmd/pyroscope/help-all.txt.tmpl

```diff
@@ -4,13 +4,137 @@ Usage of ./pyroscope:
+  -compactor.ring.etcd.tls-key-path string
+    	Path to the key for the client certificate. Also requires the client certificate to be configured.
+  -compactor.ring.etcd.tls-min-version string
+    	Override the default minimum TLS version. Allowed values: VersionTLS10, VersionTLS11, VersionTLS12, VersionTLS13
+  -compactor.ring.etcd.tls-server-name string
+    	Override the expected name on the server certificate.
+  -compactor.ring.etcd.username string
+    	Etcd username.
+  -compactor.ring.heartbeat-period duration
+    	Period at which to heartbeat to the ring. 0 = disabled. (default 15s)
+  -compactor.ring.heartbeat-timeout duration
+    	The heartbeat timeout after which compactors are considered unhealthy within the ring. 0 = never (timeout disabled). (default 1m0s)
+  -compactor.ring.instance-addr string
+    	IP address to advertise in the ring. Default is auto-detected.
+  -compactor.ring.instance-enable-ipv6
+    	Enable using a IPv6 instance address. (default false)
+  -compactor.ring.instance-id string
+    	Instance ID to register in the ring. (default "<hostname>")
+  -compactor.ring.instance-interface-names string
+    	List of network interface names to look up when finding the instance IP address. (default [<private network interfaces>])
+  -compactor.ring.instance-port int
+    	Port to advertise in the ring (defaults to -server.http-listen-port).
+  -compactor.ring.multi.mirror-enabled
+    	Mirror writes to secondary store.
+  -compactor.ring.multi.mirror-timeout duration
+    	Timeout for storing value to secondary store. (default 2s)
+  -compactor.ring.multi.primary string
+    	Primary backend storage used by multi-client.
+  -compactor.ring.multi.secondary string
+    	Secondary backend storage used by multi-client.
+  -compactor.ring.prefix string
+    	The prefix for the keys in the store. Should end with a /. (default "collectors/")
+  -compactor.ring.store string
+    	Backend storage to use for the ring. Supported values are: consul, etcd, inmemory, memberlist, multi. (default "memberlist")
+  -compactor.ring.wait-active-instance-timeout duration
+    	Timeout for waiting on compactor to become ACTIVE in the ring. (default 10m0s)
+  -compactor.ring.wait-stability-max-duration duration
+    	Maximum time to wait for ring stability at startup. If the compactor ring keeps changing after this period of time, the compactor will start anyway. (default 5m0s)
+  -compactor.ring.wait-stability-min-duration duration
+    	Minimum time to wait for ring stability at startup. 0 to disable.
+  -compactor.split-and-merge-shards int
+    	The number of shards to use when splitting blocks. 0 to disable splitting.
+  -compactor.split-groups int
+    	Number of groups that blocks for splitting should be grouped into. Each group of blocks is then split separately. Number of output split shards is controlled by -compactor.split-and-merge-shards. (default 1)
+  -compactor.tenant-cleanup-delay duration
```

Comment:
I don't think we support tenant deleteion or do we?

Reply:
Nope we don't yet

Reply:
Commenting it.